doc/specs/top_10.md

```diff
@@ -0,0 +1,106 @@
+# Top 10
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+
+## Motivation
+
+Users are notified of the top 10 most popular albums on the website. Users can view the list in a homepage widget/module. Users can see the most popular albums by past day, past week, past month, and all-time.
```

Comment:
The homepage mockup only had albums and I didn't see an option at least from their to modify it, but I suppose it could be anything.

I will remove the music terminology when I get a chance, sorry!